cmds/replay.c

```diff
@@ -1077,6 +1101,13 @@ int command_replay(int argc, char *argv[], struct opts *opts)
 	if (!opts->flat && peek_rstack(&handle, &task) == 0)
 		print_header(&output_fields, "#", 1);
 
+	if (out_color != COLOR_ON) {
+		paren_open  = "(";
+		paren_close = ")";
+		args_comma  = ", ";
+		args_retval = " = ";
+	}
+
```

Comment:
I think it's better making it a function to set those colors and call it in the common code (like in uftrace.c).  As you know, it's used in dump command too and record command doesn't care the color since it won't call `get_argspec_string()` anyway.

Reply:
Done it. Thanks!

Reply:
Why not doing it in `setup_color()`?